### PR TITLE
adding spark_job_processor

### PR DESCRIPTION
**Pull request by LaizaAngrest**

### CODE REVIEW

**Review by AvivVegh — APPROVED** [review]

File: spark_job_processor/processor.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    'remote_bytes_read': 0,
+    'local_bytes_read': 0,
+    'shuffle_bytes_read': 0,
+    'shuffle_bytes_written': 0,
+    'executor_start_time': None,
+    'executor_end_time': None,
+    'executor_cpu_time': 0,
+    'jvm_peak_memory': 0,
+    'python_peak_memory': 0,
+    'other_peak_memory': 0}
```
Comment (AvivVegh) [inline spark_job_processor/processor.py]: json } should be in new line

File: spark_job_processor/processor.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    'total_memory_per_executor': 0.0,
+    'total_num_of_cores': 0,
+    'total_bytes_read': 0,
+    'total_bytes_written': 0,
+    'total_shuffle_bytes_read': 0,
+    'total_shuffle_bytes_written': 0,
+    'total_cpu_time_used': 0,
+    'total_cpu_uptime': 0,
+    'cpu_usage': 0.0,
+    'max_memory_usage': 0.0}
```
Comment (AvivVegh) [inline spark_job_processor/processor.py]: json } should be in new line

File: spark_job_processor/processor.py (hunk 1 above)
Comment (LaizaAngrest) [reply spark_job_processor/processor.py]: done

File: spark_job_processor/processor.py (hunk 2 above)
Comment (LaizaAngrest) [reply spark_job_processor/processor.py]: done

**Pull request merged**